client/src/components/PageExplorer/PageExplorer.scss

```diff
@@ -61,10 +61,14 @@ $explorer-header-horizontal-padding: 10px;
 .c-page-explorer__header__title__inner {
   @apply w-flex;
   padding: 1em $explorer-header-horizontal-padding;
+  padding-bottom: 1rem;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 
+  outline: 1px solid #24184d;
```

Comment:
not sure if this line would be needed - there should be a default focus outline when things are focused only for all focusable targets.

Reply:
Okay, I'll check if the default focus outline works...

Reply:
No @lb- I could not find any default focus outline there...

Reply:
ok - will have to take a look with it running before I understand why this is needed.